src/components/bulk-email-tool/BulkEmailTool.jsx

```diff
@@ -7,10 +7,11 @@ import { Container } from '@openedx/paragon';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import BulkEmailTaskManager from './bulk-email-task-manager/BulkEmailTaskManager';
 import NavigationTabs from '../navigation-tabs/NavigationTabs';
-import BulkEmailForm from './bulk-email-form';
+import BuildEmailFormExtensible from './bulk-email-form/BuildEmailFormExtensible';
```

Comment:
@johnvente could you explain why both `BulkEmailForm` *and* `BuildEmailFormExtensible` exist in this PR? In my local testing I was able to delete `BulkEmailForm` and the page still worked (which I expected because it's only being consumed here https://github.com/search?q=repo%3Aopenedx%2Ffrontend-app-communications%20bulkemailform&type=code). I'm not sure if I'm missing something or if `BulkEmailForm` is just unused code at this point.

Reply:
@brian-smith-tcril At this point `BulkEmailForm` is unused code should I remove it?

Reply:
That would be perfect! Thank you!

Reply:
@brian-smith-tcril Done!